gyp_xwalk

```diff
@@ -351,6 +351,9 @@ if __name__ == '__main__':
     args.append('-Dnotifications=1')
     args.append('-Drelease_unwind_tables=0')
 
+  if sys.platform == 'win32':
```

Comment:
I tried to test both cases today, but even if I don't change anything in this pull request I still can't get the video from the URL you mentioned to work, and I get a message saying the Widevine CDM is not available. Yes, I'm passing `-Dmediacodecs_EULA=1` to the build.

`widevine_cdm_component_installer.cc` contains references to some very specific paths that may be causing this, but since there is no documentation it is not clear if that's the case.

Do the Widevine DLLs need to be in some special directory for things to work?